phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -221,25 +235,25 @@ private void checkOldIndexingCoprocessors(String [] indexList, String [] tableLi
         }
     }
 
-    @Parameters(name ="IndexUpgradeToolIT_mutable={0},upgrade={1},isNamespaceEnabled={2}")
+    @Parameters(name ="IndexUpgradeToolIT_mutable={0},upgrade={1},isNamespaceEnabled={2},isDisabledIndex={3}")
     public static Collection<Object[]> data() {
-        return Arrays.asList(new Object[][] {
-            {false, false, true},
-            {true, false, true},
-            {false, true, true},
-            {true, true, true},
-            {false, false, false},
-            {true, false, false},
-            {false, true, false},
-            {true, true, false}
-        });
+        return Arrays.asList(new Object[][] { { false, false, true, true },
+                { false, false, true, false }, { true, false, true, true },
```

Comment:
how long do all of these take to run now? Are there combinations that are redundant (or orthogonal and can be tested by a single test)?

Reply:
+1 .. The test matrix keeps getting doubled for every variant. Perhaps mix these up with some random combination of isNamespaceEnbaled.

Reply:
What do you think @kadirozde ? Should I remove the combination and make just 1 index disabled?

Reply:
The test already takes 20 min so we should not increase its time any further. Also, we can eliminate some of the existing combinations to reduce the test time as 20 min is already too long for this test.